pages/extended_search.rst

```diff
@@ -0,0 +1,152 @@
+.. _extended_search:
+
+***************
+Extended Search
+***************
+
+Views
+=====
+A views contains a set of queries. Each query has a collection of widgets
+which display messages and charts depending on the search string entered
+in the search bar and the selected time range. A view can be saved with
+a name and that way reused and enhanced later on. For a better work
```

Comment:
Suggestion: `a name to keep the current progress and continue later on. Saved views can be shared, so other people can use them in their processes.`